src/it/java/com/google/checkstyle/test/chapter4formatting/rule42blockindentation/BlockIndentation2SpacesTest.java

```diff
@@ -100,4 +100,9 @@ public void testCorrectAnnotationArrayInit() throws Exception {
         verifyWithConfigParser(MODULES, filePath);
     }
 
+    @Test
+    public void tetsFastMatcher() throws Exception {
+        final String filePath = getPath("InputFastMatcher.java");
+        verifyWithConfigParser(MODULES, filePath);
+    }
```

Comment:
`InputFastMatcher.java` file was already present but was never tested, so created test for it